en/protocol/landing_target.md

```diff
@@ -0,0 +1,71 @@
+
+Different systems may support either (or presumably both) sets of fields. 
+These are discussed below.
+
+
+## Target Relative to Captured Image {#camera_image_relative}
+
+The [LANDING_TARGET](../messages/common.md#LANDING_TARGET) fields that are relative to a captured image are shown below:
+
+Field Name | Type | Units | Values | Description
+--- | --- | --- | --- | ---
+angle_x | float | rad | | X-axis angular offset of the target from the center of the image
+angle_y | float | rad | | Y-axis angular offset of the target from the center of the image
+distance | float | m | | Distance to the target from the vehicle
+size_x | float | rad | | Size of target along x-axis
+size_y | float | rad | | Size of target along y-axis
+
+The positional information can be interpreted as described below.
+
+Imagine a ray pointing from the camera's principal point to the target. 
+The x angle (`angle_x`) is the angle spanned by that ray and the x-axis of the image plane. The same holds for the y angle (`angle_y`).
+In other words, the x angle is a function of the x pixel coordinate of the target (denoted by *u*̅in the image below), the y angle is a function of the y pixel coordinate (denoted *v*  in the image below).
+Using the angle rather than *u̅/v̅* pixel coordinates has the advantage that the effect of the camera lens is already accounted for. 
+Otherwise the receiver of the message would need to know the camera field of view etc.
+
+![Landing Target Fields](../../assets/protocols/landing_target_definitions.png)
+
+The sizes in x and y direction are analogous (`size_x`/`size_y`). 
+They describe the angle between the smallest and biggest pixel in x/y direction respectively of the target as seen in the image.
+
+> **Tip** ArduPilot supports messages with these fields only (at time of writing). 
+  The MAVLink 2 extension fields are ignored.
+
+
+## Target as Position/Quaternion (MAVLink 2 and later) {#positional}
+
+The message fields that are used to providing target as a position/quaternion are shown below.
+Their use is obvious.
+
+Field Name | Type | Units | Values | Description
+--- | --- | --- | --- | ---
+frame | uint8_t | | [MAV_FRAME](../messages/common.md#MAV_FRAME) | Coordinate frame used for following fields.
+x | float | m | | X Position of the landing target on `MAV_FRAME`
+y | float | m | | Y Position of the landing target on `MAV_FRAME`
+z | float | m | | Z Position of the landing target on `MAV_FRAME`
+q | float[4] | | | Quaternion of landing target orientation (w, x, y, z order, zero-rotation is 1, 0, 0, 0)
+type | uint8_t | | [LANDING_TARGET_TYPE](../messages/common.md#LANDING_TARGET_TYPE) | Type of landing target
+position_valid | uint8_t | | | Boolean indicating known position (1) or default unknown position (0), for validation of positioning of the landing target
```

Comment:
What is `position_valid` for/how is it used?

Reply:
Fair question, PX4 ignores it: 

https://github.com/PX4/Firmware/blob/0c5725d45ccfcccca7df3409693c6ac9a7e0bfef/src/modules/mavlink/mavlink_receiver.cpp#L2231

Perhaps the idea is that if `position_valid` use `x, y, z`, otherwise use the angle data. Currently not really needed because both autopilots only support one of the two. But i suppose an autopilot that supports both would use that to check which data was filled by the sender.

Reply:
That makes sense; any idea who might know for sure?

Reply:
My best guess would again be @TSC21, he's the author of this mavlink message: https://github.com/mavlink/mavlink/commit/c19f340b80702ba0398c4487f4db997e7396a889

Reply:
This was requested in order to set when the position being sent is a valid position coming from the sensor. By the time this was added, it wasn't considered using NaN's on the fields instead. But the idea is basically to avoid the data being used on the FCU when it's actually not a valid measurement.

Reply:
> Fair question, PX4 ignores it:

It shouldn't actually be ignored on the PX4 side, so that must be changed on the mean time. You don't want the FCU to process data that is actually faulty. Or that or we just drop the field on the Mavlink side and replace it with NaN values on the position.

Reply:
Thanks @TSC21. Perhaps stupid, but why send the message at all if the contained information is invalid?

Reply:
https://github.com/mavlink/mavlink/pull/744#issuecomment-312106517

Reply:
Awesome. Thanks.